sdk/src/events/pollingLogProvider.ts

```diff
@@ -48,7 +48,9 @@ export class PollingLogProvider implements LogProvider {
 					this.mostRecentSeenTx,
 					// If skipping history, only fetch one log back, not the maximum amount available
 					skipHistory && this.firstFetch ? 1 : undefined,
-					this.batchSize
+					this.batchSize,
+					100_000,
```

Comment:
is 100,000 really intended here? the default is 2 so this seems excessive

Reply:
idea was that there would be no discernable changes for this stuff at the moment, because we're adding these changes primarily to handle sweepstakes event fetching